CRM/Core/BAO/Dashboard.php

```diff
@@ -189,17 +176,10 @@ public static function checkPermission($permissions, $operator) {
         }
       }
 
-      if (!$showDashlet && !$hasPermission) {
-        return FALSE;
-      }
```

Comment:
Wow, a triple negative! I wonder if there's a simpler way to write that?

Reply:
Hehe. Seems there isn't not a way to avoid it.

Reply:
I wouldn't be unsurprised if we couldn't simplify it.